sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_train_labeled_model_async.py

```diff
@@ -0,0 +1,80 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_train_labeled_model_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to train a model with labeled data. See sample_recognize_custom_forms_async.py
+    to recognize forms with your custom model.
+USAGE:
+    python sample_train_labeled_model_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+    3) CONTAINER_SAS_URL - The shared access signature (SAS) Url of your Azure Blob Storage container with your labeled data.
+                      See https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/quickstarts/label-tool#connect-to-the-sample-labeling-tool
+                      for more detailed descriptions on how to get it.
+"""
+
+import os
+import asyncio
+
+
+class TrainLabeledModelSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+    container_sas_url = os.environ["CONTAINER_SAS_URL"]
+
+    async def train_labeled_model(self):
+        # [START create_form_training_client_async]
+        from azure.ai.formrecognizer.aio import FormTrainingClient
+        from azure.core.credentials import AzureKeyCredential
+
+        form_training_client = FormTrainingClient(
+            endpoint=self.endpoint, credential=AzureKeyCredential(self.key)
+        )
+        # [END create_form_training_client_async]
+        async with form_training_client:
+
+            model = await form_training_client.training(self.container_sas_url, use_labels=True)
```

Comment:
A note on the name of this function - `training` made sense when this was prefixed by `begin_`, but now that it's not, would it be better called `train_model`?

Reply:
@kristapratico is going to make a PR for this, thanks!

Reply:
#10929 Thanks Anne! We've been trying to come up with a better name for this method.